### DISCUSSION TITLE
Multiple values for f_burst

### DISCUSSION THREAD

**Discussion by ceband** [mboquien/cigale#86] [Q&A] [answered]
Hi,
I am trying to fit a burst with various mass fractions for the sfhdelayed SFH. However, when i input more than one value for f_burst I get two errors: UnpicklingError: pickle data was truncated and OSError: [Errno 22] Invalid argument.  I was wondering what the issue could be and how I could make this work. 
I am attaching the ini file and full traceback, if that is of any help. Thank you!
[pcigale.txt](https://github.com/mboquien/cigale/files/15171484/pcigale.txt)
[traceback.txt](https://github.com/mboquien/cigale/files/15171485/traceback.txt)

**Comment by mboquien** [comment, marked as answer]
That is very odd. I do not remember seeing such an error. At first glance it would seem that the database is corrupted (for instance lacking disk space when building CIGALE). Could you try to rebuild it from scratch? Also could you set `cores = 1`  and see what the backtrace says in that case?

**Reply by ceband** [reply]
Thank you for the prompt reply. Setting cores to 1 solves it
